governance/pyth_staking_sdk/src/utils/pool.ts

```diff
@@ -0,0 +1,34 @@
+import { PublicKey } from "@solana/web3.js";
+
+import type { PoolDataAccount, PublisherData } from "../types";
+
+export const extractPublisherData = (
+  poolData: PoolDataAccount,
+): PublisherData => {
+  return poolData.publishers
+    .filter((publisher) => !publisher.equals(PublicKey.default))
```

Comment:
This feels wrong if there were gaps in the publishers (which can't happen in the program though)

Reply:
yes there are a lot of assumptions based on that.